src/config.js

```diff
@@ -134,6 +134,18 @@ function createToken(username, password) {
   return Buffer.from(`${username}:${password}`).toString('base64')
 }
 
+export async function requireLogin() {
```

Comment:
Added this require login function to prompt login if the user hasn't authenticated yet.